src/main/java/nopol_examples/nopol_example_3/NopolExample.java

```diff
@@ -9,11 +9,10 @@ public boolean isOddNumber(int a) {
 		int tmp = (a-1)%2;
 		int foo = 0;
 		
-		if (tmp != 0) {
+		if (tmp < 0) { 
 			return true;
```

Comment:
This line (13) has been identified with a suspiciousness value of 26.73%.

<details>
     <summary>Failing tests that cover this line</summary>

- `nopol_examples.nopol_example_3.NopolExampleTest#test5`
- `nopol_examples.nopol_example_3.NopolExampleTest#test9`
</details>